build/docker-compose.yaml

```diff
@@ -55,23 +58,23 @@ services:
 
   snapshot-cmake: &snapshot-cmake
     <<: *build-setup
-    command: bash -c 'if [ -f CMakeLists.txt ]; then mkdir -p "$${BUILD_DIR}" && cd "$${BUILD_DIR}" && cmake .. && make -j "$${MAKEJOBS}"; fi'
+    command: bash -c 'if [ -f CMakeLists.txt ]; then mkdir -p "$${BUILD_DIR}" && cd "$${BUILD_DIR}" && cmake -DINSTALL_TARGET=RPM -DFDB_RELEASE=0 -DUSE_VALGRIND=0 /__this_is_some_very_long_name_dir_that_is_needed_to_fix_a_bug_with_cpack__/foundationdb && make -j "$${MAKEJOBS}" packages preinstall && cpack; fi'
```

Comment:
I don't think these cmake arguments are necessary. I don't know what `INSTALL_TARGET` does (this is not in the source code). `FDB_RELEASE` and `USE_VALGRING` are set to `OFF` by default - so passing them here doesn't do anything.

Reply:
Use only to allow others to easily see some of the major build options